public/terms.html

```diff
@@ -0,0 +1,419 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Hands-Down!</title>
+
+    <meta name="msapplication-TileColor" content="#C4C4C4" />
+    <meta name="msapplication-config" content="/browserconfig.xml" />
+    <meta name="theme-color" content="#C4C4C4" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta
+      name="description"
+      content="Slow the spread of COVID-19 by discouraging people from touching their face"
+    />
+    <meta
+      name="keywords"
+      content="touch, hands, down, face, collection, covid-19"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8" />
+
+    <link rel="preconnect" href="https://cdn.jsdelivr.net" />
+    <link rel="prefetch" href="dev.html" />
+
+    <!-- FavIcons -->
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/assets/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="/assets/favicon-32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="192x192"
+      href="/assets/android-chrome-192.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/assets/favicon-16.png"
+    />
+    <link rel="manifest" href="/site.webmanifest" />
+    <link
+      rel="mask-icon"
+      href="/assets/safari-pinned-tab.svg"
+      color="#C4C4C4"
+    />
+    <link
+      rel="shortcut icon"
+      sizes="16x16 24x24 32x32 48x48 64x64"
+      href="/favicon.ico"
+    />
+  </head>
+
+  <body>
+    <header>
+      <h3><strong>Hands Down!</strong></h3>
+    </header>
+    <div class="container justify">
+      <div class="row">
+        <div class="u-max-full-width">
+          <h4>Terms of Service</h4>
+
+          <p>
+            Your access to and use of
+            <a href="https://handsdown.dev">handsdown.dev</a> is subject to the
+            following terms and conditions and all applicable laws.
+          </p>
+
+          <h5>Acceptance of Terms.</h5>
+          <p>
+            The Hands Down project (or "us" or "we") welcomes you to
```

Comment:
I wasn't sure what to name the _"organisation" so I went with "The Hands Down project".

Reply:
Yeah that should be sufficient!